src/less/forms.less

```diff
@@ -800,6 +801,97 @@ input[type='checkbox']:focus {
   }
 }
 
+// add other attr to input to identify it in css file, similar to checked
+
+.rhinoslidingradio__input {
+  &:checked + .rhinoslidingradio__label::after {
+    transform-origin: center center;
+    transform: scale(1.5, 1.5);
+    transition: all 0.25s @global-trans-timing-cubic;
+  }
+  &:checked + .rhinoslidingradio__label.u-text-danger {
+    &::after {
+      border: 2px solid @global-color-danger;
+    }
+    &::before {
+      transform: scale(1, 1);
+      background-color: @global-color-danger;
+      transition: all 0.25s @global-trans-timing-cubic;
+    }
+  }
+  &:checked + .rhinoslidingradio__label.u-text-warning {
+    &::after {
+      border: 2px solid @global-color-warning;
+    }
+    &::before {
+      transform: scale(1, 1);
+      background-color: @global-color-warning;
+      transition: all 0.25s @global-trans-timing-cubic;
+    }
+  }
+  &:checked + .rhinoslidingradio__label.u-text-secondary {
+    &::after {
+      border: 2px solid @global-color-secondary;
+    }
+    &::before {
+      transform: scale(1, 1);
+      background-color: @global-color-secondary;
+      transition: all 0.25s @global-trans-timing-cubic;
+    }
+  }
+}
+
+.rhinoslidingradio__label {
+  padding: 0 0 1rem 0;
+  margin-bottom: 0;
+  cursor: pointer;
+  width: 100%;
+  text-align: center;
+
+  &::after {
+    background-color: @global-color-white;
+    border: 1px solid @global-border-color;
+    border-radius: 50%;
+    content: '';
+    display: inline-block;
+    height: 1rem;
+    left: 50%;
+    margin-left: -0.5rem;
+    position: absolute;
+    speak: none;
+    bottom: -.6rem;
+    transform-origin: center center;
+    transform: scale(1, 1);
+    width: 1rem;
+  }
+
+  &::before {
+    transform-origin: center center;
+    box-sizing: content-box;
+    height: 3px;
+    width: 100%;
+    transform: scale(0, 0);
+    content: '';
+    position: absolute;
+    bottom: -2px;
+    left: 0;
+  }
+}
+
+.rhinoslidingradio__container {
+  width: 30rem;
```

Comment:
This attribute made me wonder how we make this component re-usable. If I wanted 2 options for example, or 4. How would that work. I know it is not crucial to the current feature but if we ship as is, I suspect it will be hard to find the time to update later. 

Could we pass props for each of the options so that they attached the color classes to each? In that case we could use all existing color variations as well, not just red, yellow, green. 

Would like yo hear your thoughts.

Reply:
👍 removed per our conversation